docs/tutorial.rst

```diff
@@ -242,5 +242,151 @@ effective population size from the **TODO add crossref to catalog section**
+the terminal.
+In this tutorial, we will use the model of African-American admixture from
+`2011 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.  
+From the help output (or the :ref:`Catalog <sec_catalog>`),
+we can see that this model has id ``america``,
+and allows samples to be drawn from 4 contemporary populations representing African,
+European, Asian and African-American groups.
+
+Using the ``--help-genetic-maps`` option, we can also see what recombination maps
+are available:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-genetic-maps
+
+Let's go with ``HapmapII_GRCh37``.
+The next command simulates 4 samples of chromosome 1 from each of the four
+populations, and saves the output to a file called ``afr-america-chr1.trees``.
+For the purposes of this tutorial, we'll also specify a random seed using the
+``-s`` option.
+(Note: This took around 8 minutes to run on a laptop.)
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr1 -o afr-america-chr1.trees -s 13 -g HapmapII_GRCh37\
+    --model america 4 4 4 4
+
+We should now have a file called ``afr-america-chr1.trees``.
+Our work with ``stdpopsim`` is done; we'll now switch to a Python console and import
+the ``tskit`` package to load and analyse this simulated tree sequence file.
+
+.. code-block:: python
+
+    >>> import tskit
+    >>> ts = tskit.load(afr-america-chr1.trees)
+
+Recall that `genetic divergence` is the probability that two randomly sampled
+chromosomes differ at a nucleotide base.
+For a given pair of populations, a pair-specific divergence value is obtained
+by randomly sampling one chromosome from each population.
+These quantities can be estimated directly from our sample using tskit's
+inbuilt `TreeSequence.diversity <https://tskit.readthedocs.io/en/latest/python-api.html#tskit.TreeSequence.divergence>`_ method.
+
+The documentation for this method specifies the following required input:
+
+    sample_sets (list) – A list of lists of Node IDs, specifying the groups of individuals to compute the statistic with.
+
+    indexes (list) – A list of 2-tuples, or None.
+
+Let's think about what these sample sets and indices are in this example,
+and how we can obtain them with Python commands.
+In our case, the groups of individuals correspond to the lists 
+of sample chromosomes (nodes) from each separate population.
+We can obtain the necessary list of lists like this:
+
+.. code-block:: python
+
+    >>> sample_list = []
+    >>> for pop in range(0, ts.num_populations):
+    ...     sample_list.append(ts.samples(pop).tolist())
+    >>> print(sample_list)
+    [[0, 1, 2, 3], [4, 5, 6, 7], [8, 9, 10, 11], [12, 13, 14, 15]]
+
+Note that the samples with node IDs 0 - 3 are from population 0,
+samples with node IDs 4 - 7 are from population 1 and so on. 
+
+The indexes are all pairs of integer indices corresponding to the populations
+that we wish to compare.
+We can do this quickly with the ``itertools`` package:
```

Comment:
Probably better to call it the itertools module, since it's part of the standard library (package implies it's external)